src/barnetilsyn/BarnetilsynApp.tsx

```diff
@@ -86,10 +93,16 @@ const BarnetilsynApp = () => {
       hentMellomlagretBarnetilsyn(),
     ])
       .then(() => settFetching(false))
-      .catch(() => settFetching(false));
+      .catch(() => settFetching(false))
     // eslint-disable-next-line
   }, []);
 
+  useEffect(() => {
+    if (toggles[ToggleName.hentBarnetilsynSøknad]) {
```

Comment:
Hva skjer om denne trigges før fetching blir false? 🤔 
Bør vi ha `if(fetching === false && toggles && toggles[...])` ?

Reply:
Forrige søknad hentes ikke dersom toggles ikke er ferdig hentet, fordi toggle er undefined. Men useEffect blir gjort på nytt når fetching er ferdig og satt til false, så tenker det er nok?
Så hvertfall ut til å funke i preprod

Reply:
hvis toggles er undefined så krasjer jo appen med `Cannot read properties of undefined` ?

Reply:
Glem det - toggles er aldri undefined - den er jo `{}` tomt objekt :)